src/Components/Endpoints/src/Rendering/EndpointHtmlRenderer.cs

```diff
@@ -83,7 +83,11 @@ internal async Task InitializeStandardComponentServicesAsync(
         IFormCollection? form = null)
     {
         var navigationManager = httpContext.RequestServices.GetRequiredService<NavigationManager>();
-        ((IHostEnvironmentNavigationManager)navigationManager)?.Initialize(GetContextBaseUri(httpContext.Request), GetFullUri(httpContext.Request), OnNavigateTo);
+        ((IHostEnvironmentNavigationManager)navigationManager)?.Initialize(GetContextBaseUri(httpContext.Request), GetFullUri(httpContext.Request), uri => 
```

Comment:
Such overload for Initialize does not exist. Check out https://github.com/dotnet/aspnetcore/blob/ee050bd56bdf2b653d4bad75f26ba8802a4f58fa/src/Components/Components/src/Routing/IHostEnvironmentNavigationManager.cs#L25. We want to use `Initialize(string baseUri, string uri, Func<string, Task> onNavigateTo)`